modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc

```diff
@@ -143,22 +143,31 @@ class NDB_Menu_Filter_Form_resolved_violations extends NDB_Menu_Filter_Form
         }
 
         $this->columns = array(
-            'v.PatientName',
-            'v.Site',
-            'v.TimeRun',
-            'v.MincFile',
+            'v.Resolved as Resolution_status',
+            'v.PatientName as PatientName',
+            'site.Name as Site',
+            'v.TimeRun as TimeRun',
+            'v.MincFile as MincFile',
             'v.Series_Description as Series_Description_Or_Scan_Type',
-            'v.Problem',
-            'v.SeriesUID',
-            'v.hash',
-            'v.join_id',
-            'v.Resolved'
+            'v.Problem as Problem',
+            'v.SeriesUID as SeriesUID',
+            'v.hash as Hash',
+            'v.join_id as JoinID',
+            'v.Site as SiteID'
         );
+        $this->headers  = array('Resolution_status','PatientName','Site',
+                        'TimeRun','MincFile','Series_Description_Or_Scan_Type',
+                        'Problem','SeriesUID','Hash','JoinID','SiteID');
         if ($useProjects) {
-            array_splice($this->columns, 1, 0, 'v.Project');
-            array_splice($this->columns, 2, 0, 'v.Subproject');
+            array_splice($this->columns, 1, 0, 'v.Project as Project');
```

Comment:
The original order was not preserved + need to show Project/Subproject names instead of ID

**Before:**
![image](https://cloud.githubusercontent.com/assets/6627543/21360083/7c7228b0-c6ac-11e6-933d-46049376e6b1.png)

**After:**
![image](https://cloud.githubusercontent.com/assets/6627543/21360115/9d4db676-c6ac-11e6-98c3-7715082037a4.png)